arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/Resource.kt

```diff
@@ -761,3 +763,13 @@ private suspend fun List<suspend (ExitCase) -> Unit>.cancelAll(
   if (first != null) Platform.composeErrors(NonEmptyList(first, it))
   else Platform.composeErrors(it)
 }, { first })
+
+/**
+ * binds and emits [A] of the resource
+ */
+public fun <A> Resource<A>.asFlow(): Flow<A> =
+  flow {
+    arrow.fx.coroutines.continuations.resource {
+      emit(bind())
+    }
+  }
```

Comment:
This implementation is not correct, since now you return `Resource` to `flow { }` which is then never invoked.

```suggestion
  flow {
    use {
      emit(it)
    }
  }
```